src/Main.ts

```diff
@@ -118,7 +120,9 @@ Object.freeze(window.tsParticles);
  * @param tagId the particles container element id
  * @param params the options object to initialize the [[Container]]
  */
-window.particlesJS = (tagId: string, params: RecursivePartial<IOptions>) => ParticlesJS.load(tagId, params);
+window.particlesJS = function (tagId: string, params: RecursivePartial<IOptions>) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)